data/json/itemgroups/science_and_tech.json

```diff
@@ -307,98 +278,157 @@
-      [ "iv_mutagen", 6 ],
-      [ "iv_mutagen_elfa", 2 ],
-      [ "iv_mutagen_alpha", 2 ],
-      [ "iv_mutagen_raptor", 2 ],
-      [ "iv_mutagen_medical", 2 ],
-      [ "iv_mutagen_chimera", 2 ],
-      [ "iv_mutagen_plant", 2 ],
-      [ "iv_mutagen_insect", 2 ],
-      [ "iv_mutagen_spider", 2 ],
-      [ "iv_mutagen_snail", 2 ],
-      [ "iv_mutagen_batrachian", 2 ],
-      [ "iv_mutagen_fish", 2 ],
-      [ "iv_mutagen_slime", 2 ],
-      [ "iv_mutagen_rat", 2 ],
-      [ "iv_mutagen_beast", 2 ],
-      [ "iv_mutagen_cattle", 2 ],
-      [ "iv_mutagen_cephalopod", 2 ],
-      [ "iv_mutagen_bird", 2 ],
-      [ "iv_mutagen_lizard", 2 ],
-      [ "iv_mutagen_troglobite", 2 ],
-      [ "iv_mutagen_ursine", 2 ],
-      [ "iv_mutagen_feline", 2 ],
-      [ "iv_mutagen_lupine", 2 ],
-      [ "iv_mutagen_rabbit", 2 ],
-      [ "iv_mutagen_crustacean", 2 ],
-      [ "mutagen_chelator", 5 ],
-      [ "purifier", 3 ],
-      [ "iv_purifier", 8 ],
-      [ "syringe", 8 ]
+      { "group": "mutagenic_field_kit", "prob": 15 },
+      { "group": "mutagen_catalyst_kit", "prob": 35 },
+      { "group": "mutagen_primer_kit", "prob": 55 },
+      { "group": "mutagen_kit", "prob": 55 },
+      { "group": "mutagen_slurry_kit", "prob": 75 }
     ]
   },
   {
     "type": "item_group",
-    "id": "mut_iv",
-    "//": "If we ever get themed labs defined by what types of mutagens they worked on this itemgroup may need to be re-evaluated.",
+    "id": "mutagenic_field_kit",
+    "container-item": "box_small_plastic",
+    "subtype": "collection",
+    "items": [
+      {
+        "distribution": [
+          { "item": "iv_mutagen_elfa", "prob": 2 },
+          { "item": "iv_mutagen_alpha", "prob": 2 },
+          { "item": "iv_mutagen_raptor", "prob": 2 },
+          { "item": "iv_mutagen_medical", "prob": 2 },
+          { "item": "iv_mutagen_chimera", "prob": 2 },
+          { "item": "iv_mutagen_plant", "prob": 2 },
+          { "item": "iv_mutagen_insect", "prob": 2 },
+          { "item": "iv_mutagen_spider", "prob": 2 },
+          { "item": "iv_mutagen_snail", "prob": 2 },
+          { "item": "iv_mutagen_batrachian", "prob": 2 },
+          { "item": "iv_mutagen_fish", "prob": 2 },
+          { "item": "iv_mutagen_slime", "prob": 2 },
+          { "item": "iv_mutagen_rat", "prob": 2 },
+          { "item": "iv_mutagen_beast", "prob": 2 },
+          { "item": "iv_mutagen_cattle", "prob": 2 },
+          { "item": "iv_mutagen_cephalopod", "prob": 2 },
+          { "item": "iv_mutagen_bird", "prob": 2 },
+          { "item": "iv_mutagen_lizard", "prob": 2 },
+          { "item": "iv_mutagen_troglobite", "prob": 2 },
+          { "item": "iv_mutagen_ursine", "prob": 2 },
+          { "item": "iv_mutagen_feline", "prob": 2 },
+          { "item": "iv_mutagen_lupine", "prob": 2 },
+          { "item": "iv_mutagen_mouse", "prob": 2 },
+          { "item": "iv_mutagen_rabbit", "prob": 2 },
+          { "item": "iv_mutagen_crustacean", "prob": 2 }
```

Comment:
There's no purifier in this list but I believe there are snippets that include mentions of it being trialed in animals/mutants just like the normal serums. 

Believe it could make sense to add it, would also make purifier slightly more accessible which I believe would be a good idea.